src/packages/admin-ui-components/src/login/component.tsx

```diff
@@ -74,11 +80,7 @@ export const Login = ({ onLogin }: LoginProps) => {
 							Login
 						</Button>
 					</div>
-					{!!error && (
-						<div>
-							<b>Error: ${error.message}</b>
-						</div>
-					)}
+					{!!error && <Alert>Invalid username or password.</Alert>}
```

Comment:
Any reason we're not providing the error message? Because we allow pluggable auth, there's no guarantee that 'invalid username or password' is the error.

Reply:
One of the acceptance criteria is to not display if the password or username is at fault. Should I put the error.message in the console.log?

Reply:
my read of that requirement is that the backend shouldn’t tell you whether the username or the password individually were the problem, i have a feeling that’s already the case but we could check.

Reply:
You are right. The backend is already handling this. Fixed. I've updated it to display error.message.